### PR TITLE
apply 2020.06 to Covid19 PROD

### PR DESCRIPTION
**Pull request by atharvar28**

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## :warning: Services on branch
- portal

File: chicagoland.pandemicresponsecommons.org/manifest.json [hunk 1]
```diff
@@ -4,27 +4,27 @@
...
-    "indexd": "quay.io/cdis/indexd:2020.05",
-    "peregrine": "quay.io/cdis/peregrine:2020.05",
-    "pidgin": "quay.io/cdis/pidgin:2020.05",
-    "revproxy": "quay.io/cdis/nginx:1.17.6-ctds-1.0.1",
+    "fence": "quay.io/cdis/fence:2020.06",
+    "indexd": "quay.io/cdis/indexd:2020.06",
+    "peregrine": "quay.io/cdis/peregrine:2020.06",
+    "pidgin": "quay.io/cdis/pidgin:2020.06",
+    "revproxy": "quay.io/cdis/nginx:2020.06",
     "sheepdog": "quay.io/cdis/sheepdog:4.1.0",
```
Comment (paulineribeyre) [inline chicagoland.pandemicresponsecommons.org/manifest.json:16]: what about the sheep dog!

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## :warning: Services on branch
- portal

**Comment by PlanXCyborg** [comment]
# chicagoland.pandemicresponsecommons.org/manifest.json
## :warning: Services on branch
- portal

**Pull request merged**